test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,155 @@
+const { assertRevert } = require('../helpers/assertRevert');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('PausableCrowdsale', function ([owner, wallet, stranger]) {
+  const rate = 1;
+  const value = 1;
+
+  beforeEach(async function () {
+    this.token = await SimpleToken.new();
+    this.crowdsale = await PausableCrowdsale.new(rate, wallet, this.token.address);
+    await this.token.transfer(this.crowdsale.address, value);
+  });
+
+  describe('pause()', function () {
+    describe('when the sender is the crowdsale owner', function () {
+      const from = owner;
+
+      describe('when the crowdsale is unpaused', function () {
+        it('pauses the crowdsale', async function () {
+          await this.crowdsale.pause({ from });
+          const paused = await this.crowdsale.paused();
+          assert.equal(paused, true);
+        });
+
+        it('emits a Pause event', async function () {
+          const { logs } = await this.crowdsale.pause({ from });
+          assert.equal(logs.length, 1);
```

Comment:
Can you use our `expectEvent` helper here and in the rest of the suite?

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/b7d56d54714e01ad023429712eedb496361186bb/test/helpers/test/expectEvent.test.js#L74